packages/common/src/lib/common/tools.ts

```diff
@@ -3516,7 +3539,15 @@ export function getInstanceIndicatorObjects(namespace: string, createWakeup: boo
     return objs;
 }
 
-export function getLogger(log: any): Record<string, (msg: string) => void> {
+export interface Logger {
+    silly: (msg: string) => void;
+    debug: (msg: string) => void;
+    info: (msg: string) => void;
+    warn: (msg: string) => void;
+    error: (msg: string) => void;
+}
```

Comment:
I don't think we need this. `ioBroker.Logger` is already defined in the type definitions:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/bc49c2f0d802b9b83f6d3e99e0cfe0a4f7733cf1/types/iobroker/index.d.ts#L178-L192